data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,26 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Korath World Ship"
+	description "A damaged Korath world ship is stuck in the Ember Waste sectors boarding Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. After this return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Wont Someone Please Think Of The Children: done"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system 
+			near "Parca" 1
+		ship "Ra'gru Ik 618 (Crippled)" "Flumgorkor"
```

Comment:
We shouldn't use a constant name for a repeatable job...

```suggestion
		ship "Kas'lor Ik 582 (Crippled)"
```

Reply:
```suggestion
		fleet
			names "korath"
			variant
				"Ra'gru Ik 618 (Crippled)"
```
like this then, the way you had done it it would have no name